packages/core/src/Accordion/Accordion.tsx

```diff
@@ -65,18 +65,20 @@ export const HvAccordion = (props: HvAccordionProps) => {
 
   const { classes, cx } = useClasses(classesProp);
 
+  // The isOpen and setIsOpen will only work if the component is uncontrolled
```

Comment:
I don't know if I'm missing something here, but `setIsOpen` doesn't work when the component is controlled
by the user 😅
We have this piece of code inside the `useControlled` hook which prevents the value from updating:

```
 const setValueIfUncontrolled = useCallback(
    (newValue: SetStateAction<any>) => {
      if (!isControlled) {
        setValue(newValue);
      }
    },
    [isControlled],
  );
```

Reply:
The `setIsOpen` setter doesn't work, but the `isOpen` is being updated:
```tsx
// useControlled.ts:14
const value = isControlled ? controlledProp : valueState;
```

Here's an example of the controlled/uncontrolled behaving the same way:
https://stackblitz.com/edit/github-qjjtev?file=src%2FApp.tsx

Reply:
Yeah, you're right I missed that 🤦🏼‍♀️ thanks, I'll update the PR

Reply:
I'll close the PR. I was going to make the following change `onChange?.(event, !isOpen);` because the current expanded state is not returned, but I'm not sure if we should change this for now.